Resources/views/layout.html.twig

```diff
@@ -19,10 +19,7 @@ file that was distributed with this source code.
 
             {% block sonata_page_breadcrumb %}
                 <div class="row">
-                    {% if sonata_seo_context is not defined %}
-                        {% set sonata_seo_context = 'homepage' %}
-                    {% endif %}
-                    {{ sonata_block_render_event('breadcrumb', { 'context': sonata_seo_context, 'current_uri': app.request.requestUri }) }}
+                    {{ sonata_block_render_event('breadcrumb', { 'context': 'page', 'current_uri': app.request.requestUri }) }}
```

Comment:
I don't really understand why we suddenly no longer need `sonata_seo_context`. Are there any docs about this variable?

Reply:
Hey @greg0ire,
I removed it because I didnt see any info about it in the docs. I also checked my project, which has all dependencies for the pagebundle installed; I couldn't find any usage of it besides the layout.html.twig.